Gruntfile.js

```diff
@@ -67,14 +67,15 @@ module.exports = function (grunt) {
                     banner: w2ui
                 },
                 src: [
-                    'src/w2utils.js', 
-                    'src/w2grid.js', 
-                    'src/w2layout.js', 
-                    'src/w2popup.js', 
-                    'src/w2tabs.js', 
-                    'src/w2toolbar.js', 
-                    'src/w2sidebar.js', 
-                    'src/w2fields.js', 
+                    'src/w2utils.js',
+                    'src/w2grid.js',
+                    'src/w2layout.js',
+                    'src/w2popup.js',
+                    'src/w2panel.js',
+                    'src/w2tabs.js',
+                    'src/w2toolbar.js',
+                    'src/w2sidebar.js',
+                    'src/w2fields.js',
```

Comment:
w2fields is a subset of w2ui with just fields part of it. There is no need to add all into that subset.